angular/projects/admin-nrpti/src/app/mines/mines-records-rows/mines-records-table-row.component.ts

```diff
@@ -70,52 +71,7 @@ export class MinesRecordsTableRowComponent extends TableRowComponent implements
    * @memberof MinesRecordsTableRowComponent
    */
   goToDetails() {
-    switch (this.rowData._schemaName) {
```

Comment:
After working on #521 I cannot begin to express how much happiness I get from this change.